CHANGELOG.md

```diff
@@ -1,32 +1,120 @@
+    [#14694](https://github.com/rust-lang/cargo/pull/14694)
+- Deprecate `cargo verify-project`.
+  [#14736](https://github.com/rust-lang/cargo/pull/14736)
+- Add source replacement info when no matching package found during dependency resolving.
+  [#14715](https://github.com/rust-lang/cargo/pull/14715)
+- Hint for using `crates-io` when `[patch.crates.io]` found.
+  [#14700](https://github.com/rust-lang/cargo/pull/14700)
+- Normalize source paths of Cargo targets for better diagnostics.
+  [#14497](https://github.com/rust-lang/cargo/pull/14497)
+  [#14750](https://github.com/rust-lang/cargo/pull/14750)
+- Docs for optional registry JSON fields.
+- Allow registries to omit empty/default fields in index metadata JSON.
+  Due to backward compatibility, crates.io continues to emit them.
+  [#14838](https://github.com/rust-lang/cargo/pull/14838)
+  [#14839](https://github.com/rust-lang/cargo/pull/14839)
+- cargo-doc: display env vars in extra verbose mode.
+  [#14812](https://github.com/rust-lang/cargo/pull/14812)
+- cargo-fix: replace special-case handling of duplicate insert-only replacement.
+  [#14765](https://github.com/rust-lang/cargo/pull/14765)
+  [#14782](https://github.com/rust-lang/cargo/pull/14782)
+- cargo-remove: when a dependency is not found, try suggesting other dependencies
+  with similar names.
+  [#14818](https://github.com/rust-lang/cargo/pull/14818)
+- git: skip unnecessary submodule validations for fresh checkouts on Git dependencies.
+  [#14605](https://github.com/rust-lang/cargo/pull/14605)
+- git: Enhanced the error message for fetching Git dependencies when refspec not found.
+  [#14806](https://github.com/rust-lang/cargo/pull/14806)
+- git: Pass `--no-tags` by default to git CLI when `net.git-fetch-with-cli = true`.
+  [#14688](https://github.com/rust-lang/cargo/pull/14688)
 
 ### Fixed
 
+- Fixed old Cargos failing to read the newer format of dep-info in build caches.
+  [#14751](https://github.com/rust-lang/cargo/pull/14751)
+  [#14745](https://github.com/rust-lang/cargo/pull/14745)
+- Fixed rebuild detection not respecting changes in the `[env]` table.
+  [#14701](https://github.com/rust-lang/cargo/pull/14701)
+  [#14730](https://github.com/rust-lang/cargo/pull/14730)
+- cargo-fix: Added transactional semantics to `rustfix` to keep code fix in a
+  valid state when multiple suggestions contain overlapping spans.
+  [#14747](https://github.com/rust-lang/cargo/pull/14747)
+
 ### Nightly only
 
+- The unstable environment variable `CARGO_RUSTC_CURRENT_DIR` has been removed.
+  [#14799](https://github.com/rust-lang/cargo/pull/14799)
+- 🔥 A new unstable feature flag to generate a JSON schema for the manifest.
+  Useful for external tools to further validate or complete Cargo.toml.
+  ([manifest.schema.json](https://github.com/rust-lang/cargo/blob/master/crates/cargo-util-schemas/manifest.schema.json))
+  [#14683](https://github.com/rust-lang/cargo/pull/14683)
```

Comment:
The way this is written, it makes it sound like this is a feature flag on Cargo when this is a changelog entry for one of our helper libraries.  Do we normally put these here?  

We could frame this as a nightly documentation of a JSON Schema file for manifests

Reply:
Thanks for the suggestion. Made it focus on the JSON Schema file only.